group_vars/all/users.yml

```diff
@@ -80,7 +80,7 @@ users:
   singlerider:
     irc_oper:
       type: sslclientcertfp
-      pass: "{{ vault_users.singlerider.irc_operpass }}"
+      pass: "41:AE:61:33:58:25:B2:BB:30:D1:07:1A:2C:F6:A7:CB:59:A6:E6:81:16:CA:86:82:09:2F:F3:F3:3F:0D:0E:88"
```

Comment:
@RyanSquared No, that's valid, a certificate fingerprint is not a secret (unlike a password hash that can be bruteforced...)